content/docs/help/faq.md

```diff
@@ -16,5 +16,22 @@ top = false
 
 ## What is Biscuit?
 
+## How can biscuit be integrated with OAuth and OIDC?
+
+Biscuit is focused on the authorization and delegation side of API access, so it can be used as an OAuth access token. OAuth does not mandate a specific format for the access token.
+Refresh tokens work better when they are meant for a single use. In that case, a random string stored in the Authorization Server (in OAuth terms) database will fit the task.
+OIDC specifies that identity information is transmitted using JWT, so Biscuit will not be usable in their place.
+
+## Why Datalog?
+
+When searching for a suitable language to write authorization policies, it appeared that logic languages are generally a good fit. Other projects like OPA and Oso came to the same conclusion. Datalog is simple enough to be taught in a few minutes, and its implementation is straightforward, which makes it a good choice for safe authorization.
```

Comment:
why a custom language?: it can be embed in tokens and have the same behaviour on several language stacks
why a datalog variant?: being able to express rules and conditions is a good fit for authorization polices. Datalog has well-understood semantics and has been used in this domain for some time. It is also more flexible and more expressive than purely descriptive DSLs like what is available in zanzibar: biscuit aims at letting you choose the authorization model
why not an existing datalog? simple semantics, reduced dependency footprint, runtime constraints (memory used, time spent)